app/src/main/java/com/github/steroidteam/todolist/model/todo/Task.java

```diff
@@ -39,6 +42,15 @@ public String getBody() {
         return this.body;
     }
 
+    public void setLocationReminder(Location location) {
+        Objects.requireNonNull(location);
```

Comment:
If we add this condition (which I think is good, just like we do with due dates), we need another method to remove the location reminder. See `Task.removeDueDate` for reference.